Makefile

```diff
@@ -105,7 +103,7 @@ test-e2e: ##@tests Run e2e tests
 	build/env.sh go test -timeout 5m ./e2e/accounts/... -network=$(networkid)
 	build/env.sh go test -timeout 5m ./e2e/api/... -network=$(networkid)
 	build/env.sh go test -timeout 5m ./e2e/node/... -network=$(networkid)
-	build/env.sh go test -timeout 15m ./e2e/jail/... -network=$(networkid)
+	build/env.sh go test -timeout 50m ./e2e/jail/... -network=$(networkid) -v
```

Comment:
Please, remove `-v`, we don't add this for CI.

Reply:
We got error https://docs.travis-ci.com/user/common-build-problems/#Build-times-out-because-no-output-was-received becouse run test without `-v` flag.
On one hand we do `time.NewTimer(20 * time.Minute)` to wait for sync, but travis fails with error after 10 minutes without output.
I can do another thing add some output every 5 munites in `EnsureNodeSync`. Something like `still syncing`.

Reply:
Have you seen the link? https://docs.travis-ci.com/user/common-build-problems/#Build-times-out-because-no-output-was-received

Reply:
Yep, twice. This is not good solution because in travis we'll see nothing. In case of timeout developer will see nothing for travis_wait time and then the travis error that says nothing: is it test timeout or sync timeout.
May be one message every 5 minutes is better solution.